client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -51,24 +49,10 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 				</div>
 				<div className="go-mobile__app-badges">
 					{ showIosBadge && (
-						<AppsBadge
-							storeLink="https://apps.apple.com/app/apple-store/id335703880?pt=299112&ct=calypso-customer-home&mt=8"
-							storeName={ 'ios' }
-							titleText={ translate( 'Download the WordPress iOS mobile app.' ) }
-							altText={ translate( 'Apple App Store download badge' ) }
-						>
-							<img src={ appleStoreLogo } alt="" />
-						</AppsBadge>
+						<AppsBadge storeName={ 'ios' } referrer={ 'go-mobile-card' }></AppsBadge>
```

Comment:
Is there any reason why you changed the referrer from `calypso-customer-home` to `go-mobile-card`?

I guess that could affect any monitoring tool we have in place watching for the `calypso-customer-home` value.

Reply:
So... My bad... My first idea was to change the referrer from the component I created on my last PR (cards/tasks/go-mobile), in order - *actually*-  not to affect the monitoring. I was late and I guess I mixed up the components 🤦 
I'll fix it now. Should I keep the same campaign on the other card, or change it?